### DISCUSSION TITLE
Why did I get an email from GitHub and Trade Controls and no context provided?

### DISCUSSION THREAD

**Discussion by cazepeda** [community/community#125582] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I received an email this morning from `GitHub and Trade Controls` with the following message but no context provided as to why. I review my GitHub account every morning and never saw any `noticed a message` on my account. I also haven't made a purchase ever!

```
You may have noticed a message that your account was under review. Thank you for your patience while we reviewed your case. We've resolved the issue with your account and you can now continue with your GitHub purchase.
We're sorry for any inconvenience.
```

I hope to get some feedback on this. Below is screenshot of the email.

![Screenshot 2024-05-24 at 9 08 30 AM](https://github.com/community/community/assets/24512470/9b91d3cc-d04a-49ca-ac6c-b83a11d89be0)

**Comment by casmokey** [comment]
I also received the same email.
![Screenshot 2024-05-24 135944](https://github.com/community/community/assets/45880019/0a5f5e2b-0934-492e-a697-dbaf262f177f)

**Comment by mazib** [comment]
I also got this email, why?

**Comment by joaquinllopez00** [comment]
I also received this email

**Comment by sl4m** [comment]
Not sure if it helps, but it may be tied to GitHub sponsors. I noticed in the [security log](https://github.com/settings/security-log), GitHub Staff cancelled all of my sponsors (donation recipients) and I'm not sure why.
